app/models/rent.rb

```diff
@@ -252,24 +230,24 @@ def p_email
   def as_json(options = {})
     if service
       {
-          :id => self.id,
-          :title => "Service",
-          :start => self.d_from.rfc822,
-          :end => self.d_til.rfc822,
-          :status => self.comment,
-          :backgroundColor => "black",
-          :textColor => "white"
+          id: id,
+          title: 'Service',
+          start: d_from.iso8601,
+          end: d_til.iso8601,
+          status: comment,
+          backgroundColor: 'black',
+          textColor: "white"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.